project_euler/problem_145/sol1.py

```diff
@@ -13,44 +13,81 @@
 
 How many reversible numbers are there below one-billion (10^9)?
 """
+EVEN_DIGITS = [digit for digit in range(10) if digit % 2 == 0]
+ODD_DIGITS = [digit for digit in range(10) if digit % 2 == 1]
 
 
-def odd_digits(num: int) -> bool:
+def reversible_numbers(
```

Comment:
Can you explain the algorithm used by this function? If there's any references used, please mention that as well.

This would be useful for people looking at this solution and wondering what's going on :)

Reply:
Added a small comment with explanation